src/puppetlabs/puppetdb/http/handlers.clj

```diff
@@ -127,6 +127,17 @@
                        identity
                        (http/status-not-found-response "environment" environment)))))
 
+(defn producer-status
+  "Produce a response body for a single producer."
```

Comment:
I think we generally follow the Clojure docstring tense, so perhaps "Produces..."?